src/unistd.rs

```diff
@@ -737,11 +737,23 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+/// Directive that tells lseek how to reposition the offset of the file.
```

Comment:
Please reference `lseek` and `lseek64` and specify them as Markdown links to the documentation. See [here](https://doc.rust-lang.org/src/core/num/mod.rs.html#2635) for an example. You'll need to figure out the exact paths for lseek/lseek64, but the premise is the same.